### PR TITLE
[Doc] Created more detailed README.md

### PR DESCRIPTION
**Pull request by alongosz**
This PR provides a better README for the eZ Platform Admin UI Bundle.
At the time of creating it, the `1.2` is still partially maintained, hence targeting it (as it contains link to the bug tracker).

**TODO**:
- [x] Add information about issue/bug tracker.
- [x] Add build, version, license icons.
- [x] Add link to the eZ Platform installation instructions.
- [x] Add copyright and license informations.
- [x] Ask for a code review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+[![Build Status](https://img.shields.io/travis/ezsystems/ezplatform-admin-ui.svg?style=flat-square&branch=master)](https://travis-ci.org/ezsystems/ezplatform-admin-ui)
+[![Latest version](https://img.shields.io/github/release/ezsystems/ezplatform-admin-ui.svg?style=flat-square)](https://github.com/ezsystems/ezplatform-admin-ui/releases)
+[![License](https://img.shields.io/github/license/ezsystems/ezplatform-admin-ui.svg?style=flat-square)](LICENSE)
+
+This repository is dedicated to the eZ Platform Admin UI Bundle, which contains all the necessary
+parts to run the [eZ&nbsp;Platform](https://www.ezplatform.com/) administration interface (a.k.a. "back office").
+
+## Installation
+
+This package is a part of the [eZ Platform](https://github.com/ezsystems/ezplatform).
```
Comment (DominikaK) [inline README.md]:
```suggestion
This package is a part of [eZ Platform](https://github.com/ezsystems/ezplatform).
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+Please see [the online docs](https://doc.ezplatform.com/en/latest/getting_started/install_using_composer/) for the installation instructions.
```
Comment (DominikaK) [inline README.md]:
```suggestion
Please see [the online docs](https://doc.ezplatform.com/en/latest/getting_started/install_ez_platform/) for the installation instructions.
```

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by alongosz** [comment]
Thanks. Merged up to `1.3`, `1.4`, `master`.

**Pull request merged**